resources/es/cemu.po

```diff
@@ -634,15 +924,15 @@ msgstr ""
 
 #: GeneralSettings2.cpp
 msgid "Discord Presence"
-msgstr "Rich Presence de Discord"
+msgstr "Presencia enriquecida de Discord"
```

Comment:
It seems like that the official name of this feature is "Rich Presence" in Spanish too, see https://discord.com/rich-presence. Therefore I'm not sure whether its name should be translated

Reply:
Yes, I hope Discord will correct and update that term.

Reply:
My idea is that we should stick with officially translated names as much as we can. Obviously, if you deem the official name incorrect and your translation is more commonly used, I have no problems merging it.